projects/ngx-prx-styleguide/src/assets/styles/_layout.scss

```diff
@@ -40,8 +40,6 @@ main {
   }
 }
 
-article .hero > section,
-article .hero div > section,
 article .main > section {
```

Comment:
meh, I'm using this CSS right now on the production calendar. It has a hero without a toolbar, so the prx-hero doesn't work

Reply:
This style was moved into the component SCSS, so the padding this rule provides is now provided by the component internally.

Reply:
nvm, I see what you are saying. I can add this back into the global CSS, unless you can add this padding in the production calendar CSS?

Reply:
yeah I see what you're going for there as well, probably the right move. let's maybe do this? 1) put the padding in the production calendar component, but 2) could we make a ticket to separate the hero from the toolbar, or do those have to be paired? I feel like we could clean this up in the short term because we have that toolbar work happening.

Reply:
@rpeterman-gp Does eventually separating the hero banner from the hero toolbar make sense to you? The production calendar only needs the banner. They're totally separate elements in the template. Looks like a decently easy refactor to pull out the banner.

banner:
```
    <div #refHero class="hero banner" [class.orange]="orange" [class.blue]="blue">
      <section>
        <ng-content select=".hero-title"></ng-content>
      </section>
    </div>
```
toolbar:
```
    <div class="hero toolbar" [class.affix]="affixed" (window:scroll)="onScroll()">
      <section>
        <div class="info" #refInfo><ng-content select=".hero-info"></ng-content></div>
        <prx-spinner *ngIf="refInfo.children.length == 0" inverse=true></prx-spinner>
        <div class="actions" #refActions><ng-content select=".hero-actions"></ng-content></div>
      </section>
    </div>
    <div class="spacer" [class.affix]="affixed"></div>
```